lib/lotus/helpers/numbers/number_formatter.rb

```diff
@@ -0,0 +1,52 @@
+module Lotus
+  module Helpers
+    module Numbers
+      # Numbers
+      #
+      # @since x.x.x
+      class NumberFormatter
+        # NumberFormatter
+        #
+        # @since x.x.x
+        # @api private
+
+        attr_accessor :number, :delimiter, :separator, :precision
```

Comment:
If there isn't a good reason to keep those public, should we declare those as private?